File: Sistema de Control Clínico Veterinario/Sistema_Control_Clinico_Veterinario/logica/Cliente.cs
public class Cliente : Persona
{
    public int Id { get; set; }
    public string Nit { get; set; }
    public string Telefono { get; set; }
    public string Email { get; set; }
    public string Direccion { get; set; }
    public string Usuario { get; set; }
    public string Contrasena { get; set; }

    public List<Mascota> Mascotas { get; set; } = new List<Mascota>();

    public void AgregarMascota(Mascota m)
    {
        Mascotas.Add(m);
        m.Propietario = this;
    }
}

File: Sistema de Control Clínico Veterinario/bluej/Cliente.java
import java.util.ArrayList;
import java.util.List;

public class Cliente extends Persona {
    private String nit;
    private List<Mascota> mascotas;
    
    public Cliente(String nombre, String dpi, String telefono, String email, String nit) {
        super(nombre, dpi, telefono, email);
        this.nit = nit;
        this.mascotas = new ArrayList<>();
    }
    
    public void agregarMascota(Mascota m) {
        mascotas.add(m);
    }
    
    public List<Mascota> getMascotas() { return mascotas; }
    public String getNit() { return nit; }
}